<!-- test-analysis-auto -->
@claude 以下のテスト失敗を修正してください。

**失敗ジョブ**: UI Tests
**Workflow Run**: https://github.com/sy-hash/mind-echo/actions/runs/22522627474

```
UI Tests	Run UI Tests (MindEchoUITests)	2026-02-28T14:35:54.8654990Z /Users/runner/work/mind-echo/mind-echo/MindEcho/MindEchoUITests/Tests/EntryDetailUITests.swift:87: error: -[MindEchoUITests.EntryDetailUITests testSwipeToDelete_removesRecording] : XCTAssertTrue failed
UI Tests	Run UI Tests (MindEchoUITests)	2026-02-28T14:35:55.6163960Z Test Case '-[MindEchoUITests.EntryDetailUITests testSwipeToDelete_removesRecording]' failed (14.151 seconds).
UI Tests	Run UI Tests (MindEchoUITests)	2026-02-28T14:35:56.5256800Z 	 Executed 4 tests, with 1 failure (0 unexpected) in 89.053 (89.055) seconds
UI Tests	Run UI Tests (MindEchoUITests)	2026-02-28T14:36:18.1953760Z 	 Executed 3 tests, with 0 failures (0 unexpected) in 22.840 (22.841) seconds
UI Tests	Run UI Tests (MindEchoUITests)	2026-02-28T14:36:49.2294450Z 	 Executed 1 test, with 0 failures (0 unexpected) in 31.152 (31.152) seconds
UI Tests	Run UI Tests (MindEchoUITests)	2026-02-28T14:37:15.4185810Z 	 Executed 3 tests, with 0 failures (0 unexpected) in 26.149 (26.155) seconds
UI Tests	Run UI Tests (MindEchoUITests)	2026-02-28T14:38:00.3699610Z 	 Executed 4 tests, with 0 failures (0 unexpected) in 45.041 (45.045) seconds
UI Tests	Run UI Tests (MindEchoUITests)	2026-02-28T14:38:00.4435450Z 	 Executed 15 tests, with 1 failure (0 unexpected) in 214.234 (214.252) seconds
UI Tests	Run UI Tests (MindEchoUITests)	2026-02-28T14:38:00.4436430Z 	 Executed 15 tests, with 1 failure (0 unexpected) in 214.234 (214.253) seconds
```

テストが通るようにコードを修正し、ビルドが通ることも確認してください。

--- a/MindEcho/MindEcho/ViewModels/HomeViewModel.swift
+++ b/MindEcho/MindEcho/ViewModels/HomeViewModel.swift
@@ -216,8 +216,8 @@ class HomeViewModel {
         if entry.recordings.isEmpty {
             // Delete associated audio files and the entry itself
             modelContext.delete(entry)
-            fetchAllEntries()
         }
+        fetchAllEntries()
     }
 
     func exportForSharing(entry: JournalEntry) async throws -> URL {

--- a/MindEcho/MindEcho/Views/HomeView.swift
+++ b/MindEcho/MindEcho/Views/HomeView.swift
@@ -120,14 +120,32 @@ struct HomeView: View {
         let identifier = isToday
             ? "home.recordingRow.\(recording.sequenceNumber)"
             : "past.recordingRow.\(dateTag(entry!.date)).\(recording.sequenceNumber)"
+        let isCurrentlyPlaying = viewModel.playingRecordingId == recording.id && viewModel.isPlaying
 
         Button {
             transcriptionTargetRecording = recording
         } label: {
-            recordingRowContent(
-                recording: recording, isToday: isToday,
-                entry: entry)
-            .contentShape(Rectangle())
+            HStack(alignment: .center, spacing: 0) {
+                Button {
+                    if isCurrentlyPlaying {
+                        viewModel.pausePlayback()
+                    } else {
+                        viewModel.playRecording(recording)
+                    }
+                } label: {
+                    Image(systemName: isCurrentlyPlaying ? "pause.fill" : "play.fill")
+                        .frame(width: 44, height: 44)
+                }
+                .accessibilityIdentifier(
+                    isToday
+                        ? "home.playButton.\(recording.sequenceNumber)"
+                        : "past.playButton.\(dateTag(entry!.date)).\(recording.sequenceNumber)"
+                )
+                recordingRowContent(
+                    recording: recording, isToday: isToday,
+                    entry: entry)
+                .contentShape(Rectangle())
+            }
         }
         .buttonStyle(.plain)
         .accessibilityIdentifier(identifier)

--- a/MindEcho/MindEcho/Views/HomeView.swift
+++ b/MindEcho/MindEcho/Views/HomeView.swift
@@ -121,13 +121,15 @@ struct HomeView: View {
             ? "home.recordingRow.\(recording.sequenceNumber)"
             : "past.recordingRow.\(dateTag(entry!.date)).\(recording.sequenceNumber)"
 
-        recordingRowContent(
-            recording: recording, isToday: isToday,
-            entry: entry)
-        .contentShape(Rectangle())
-        .onTapGesture {
+        Button {
             transcriptionTargetRecording = recording
+        } label: {
+            recordingRowContent(
+                recording: recording, isToday: isToday,
+                entry: entry)
+            .contentShape(Rectangle())
         }
+        .buttonStyle(.plain)
         .accessibilityIdentifier(identifier)
         .swipeActions(edge: .trailing) {
             if let targetEntry = isToday ? viewModel.todayEntry : entry {

--- a/docs/ui-test-design.md
+++ b/docs/ui-test-design.md
@@ -43,6 +43,7 @@ TabView を廃止し、今日のセクションと過去の履歴セクション
 - `home.emptyState` — 録音がない場合の空状態テキスト
 - `home.recordButton` — 録音開始ボタン（画面下部固定）
 - `home.recordingRow.{n}` — 今日の録音行（n = sequenceNumber）。セル全体がタップ領域で、タップすると書き起こし詳細画面を表示
+- `home.playButton.{n}` — 今日の録音の再生/一時停止ボタン（セル左端）
 - `home.transcription.{n}` — 今日の録音セル内の書き起こしテキストプレビュー
 - `home.summary.{n}` — 今日の録音セル内の要約テキストプレビュー（要約がある場合、書き起こしプレビューより優先表示）
 - `home.deleteButton.{n}` — 今日の録音のスワイプ削除ボタン
@@ -58,6 +59,7 @@ TabView を廃止し、今日のセクションと過去の履歴セクション
 - `past.shareAudioButton.{date}` — 過去の共有メニュー内の「音声を共有」ボタン
 - `past.shareTranscriptButton.{date}` — 過去の共有メニュー内の「テキストを共有」ボタン
 - `past.recordingRow.{date}.{n}` — 過去の録音行。セル全体がタップ領域で、タップすると書き起こし詳細画面を表示
+- `past.playButton.{date}.{n}` — 過去の録音の再生/一時停止ボタン（セル左端）
 - `past.transcription.{date}.{n}` — 過去の録音セル内の書き起こしテキストプレビュー
 - `past.summary.{date}.{n}` — 過去の録音セル内の要約テキストプレビュー
 - `past.deleteButton.{date}.{n}` — 過去の録音のスワイプ削除ボタン

